src/frontend/tsx/App.tsx

```diff
@@ -10,6 +10,14 @@ import { GamePanel } from './GamePanel'
 export function App () {
   const { urbit, setUrbit, receiveChallenge, receiveGame } = useChessStore()
 
+  //
+  // State
+  // XX: Make the state more efficient with one array (or something) for prefereneces.
+  // const [preferences, setPreferences] = useState([])
+  //
+  const [pieceTheme, setPieceTheme] = useState<string>('tatiana')
+  const [boardTheme, setBoardTheme] = useState<string>('blue')
```

Comment:
On its own, this state will reset on reload/refresh, meaning that the user needs to set his visual preferences every single time he opens the chess app. As discussed in the comments on this PR, you're going to want to combine the React state with [`localStorage`](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage). [Here](https://www.freecodecamp.org/news/how-to-use-localstorage-with-react-hooks-to-set-and-get-items/) is an example guide, but you might want to find another one (there are several online).